WcaOnRails/db/migrate/20170316030443_add_big_json_to_competitions.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+class AddBigJsonToCompetitions < ActiveRecord::Migration[5.0]
+  def change
+    add_column :Competitions, :big_json, :text
```

Comment:
Yay knowledge! Things make more sense now. Still think it should be a separate table.

BTW: In case we'd consider moving someday, JSON is a native data type in Postgres.